### ISSUE TITLE
Shallow copy does not work with Django 1.9's JSONField.

### ISSUE THREAD

**Issue by prajnamort** [romgar/django-dirtyfields#77]
Use a deepcopy would fix that.

**Comment by romgar** [comment]
Good to know. I will add some postgresql specific tests! Thanks for your feedback @prajnamort !

**Comment by romgar** [comment]
@prajnamort a basic test for `JSONField` is working (#82).

Can you copy/paste some code that is not working as you expected, or even better, write a test for it?

**Comment by prajnamort** [comment]
```python
def test_dirty_json_field():
    from tests.models import TestModelWithJSONField

    tm = TestModelWithJSONField.objects.create(json_field={'data': 'dummy_data', 'lst': [1, 2, 3]})
    assert tm.get_dirty_fields() == {}

    lst = tm.json_field['lst']
    lst.append(4)
    assert tm.get_dirty_fields() == {'json_field': {'data': 'dummy_data', 'lst': [1, 2, 3]}}
```

**Comment by prajnamort** [comment]
@romgar 
In fact, this issue exists for all mutable fields (jsonfield, postgresql array, postgresql json).
Using a deepcopy would solve that ([#87](https://github.com/romgar/django-dirtyfields/pull/87))

**Comment by romgar** [comment]
I'm in holidays right now. I will come back to you in 2 weeks (also regarding the pk issue #89). Thanks for your contributions!!

**Comment by romgar** [comment]
Fixed by MR #87! Thanks @prajnamort.
